### ISSUE TITLE
Duplicate items

### ISSUE THREAD

**Issue by fnk0** [KyoheiG3/PagingView#2]
So.. What I want is a PagingView with 2 images. Without infinite scrolling.

I followed your example and made ImagePagingCell extending PagingViewCell  (basically just a UIImageView that fills the entire nib like the one in your demo). But the problem is that instead of only having 2 images I have 4, the 2 images duplicated.

Here's some code:

``` swift
// Inside viewDidLoad
self.pagingView.dataSource = self
self.pagingView.delegate = self
self.pagingView.infinite = false

// then...
func pagingView(pagingView: PagingView, numberOfItemsInSection section: Int) -> Int {
   return 2
}   

func pagingView(pagingView: PagingView, cellForItemAtIndexPath indexPath: NSIndexPath) -> PagingViewCell {
       pageControl.currentPage = indexPath.row
       let cell = pagingView.dequeueReusableCellWithReuseIdentifier("ImagePagingViewCell")
       var url : String = ""
       if indexPath.row == 0 {
          url = (self.data?.img0)!
       } else if indexPath.row == 1 {
           url = (self.data?.img1)!
       }
       let imageURL = NSURL(string: "\(Const.URL_BASE_IMAGE)\(url)")
       if let cell = cell as? ImagePageViewCell {
           cell.imageView.af_setImageWithURL(imageURL!)
       }
       return cell
}

```

Any ideas why I'm getting 4 pages instead of 2? 

Thanks

**Comment by KyoheiG3** [comment]
How about the following:

``` swift
func pagingView(pagingView: PagingView, numberOfItemsInSection section: Int) -> Int {
    return 4
}

func pagingView(pagingView: PagingView, cellForItemAtIndexPath indexPath: NSIndexPath) -> PagingViewCell {
    pageControl.currentPage = indexPath.row
    let cell = pagingView.dequeueReusableCellWithReuseIdentifier("ImagePagingViewCell")
    var url : String = ""
    if indexPath.row % 2 == 0 {
        url = (self.data?.img0)!
    } else if indexPath.row % 2 == 1 {
        url = (self.data?.img1)!
    }
    let imageURL = NSURL(string: "\(Const.URL_BASE_IMAGE)\(url)")
    if let cell = cell as? ImagePageViewCell {
        cell.imageView.af_setImageWithURL(imageURL!)
    }
    return cell
}
```

**Comment by fnk0** [comment]
I think you misunderstood what I'm asking... I only want 2 images. But for some reason I'm getting those 2 images duplicated giving me 4 images... 

It's been driving me insane since I'm specifying `numberOfItemsInSection` to 2 but I'm getting 4 for some reason :(

**Comment by KyoheiG3** [comment]
I'm sorry. I misunderstood.

All of the Cell is case of two or less, bug occurs.

**Comment by rad182** [comment]
did some workarounds. not sure if this is okay.
https://github.com/KyoheiG3/PagingView/pull/7
@KyoheiG3

**Comment by KyoheiG3** [comment]
FIx in [0.4.2](https://github.com/KyoheiG3/PagingView/tree/0.4.2).
Thank you.
